tools/azure-sdk-tools/ci_tools/environment_exclusions.py

```diff
@@ -0,0 +1,407 @@
+#!/usr/bin/env python
```

Comment:
@kristapratico I don't even know what Git is doing nowadays. I _explicitly_ ensured that I ran 

`git mv ./eng/tox/environment_exclusion_list.py ./tools/azure-sdk-tools/ci_tools/environment_exclusions.py` and it _still_ counts it as a file delete.

Actually infuriating :|

Reply:
No worries!